assets/package.json

```diff
@@ -13,7 +13,8 @@
     "alpinejs-breakpoints": "^2.1.1",
     "phoenix": "file:../deps/phoenix",
     "phoenix_html": "file:../deps/phoenix_html",
-    "phoenix_live_view": "file:../deps/phoenix_live_view"
+    "phoenix_live_view": "file:../deps/phoenix_live_view",
+    "showdown": "^2.1.0"
```

Comment:
why did you pick `showdown`. seems a bit abandoned with last commit 2y ago.

maybe try https://github.com/markedjs/marked which seems to be actively maintained and with useful hooks for modifying how things get rendered.

also not sure how good sanitization is. try:

```html
<script>alert("hello")</script>
```

and

```html
<img src="x" onerror="alert('hello')" />
```